backend/src/application.js

```diff
@@ -34,6 +34,7 @@ module.exports = function application(
   app.use(cors());
   app.use(helmet());
   app.use(bodyparser.json());
+  app.use(express.static(path.join(__dirname, 'public')));
```

Comment:
Awesome David thanks for adding this, I originally thought on only adding `expres.static('public')` but in fact we need the path reference as well, I'm glad you covered it